README.adoc

```diff
@@ -7,7 +7,16 @@ image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/
 image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml/badge.svg["Compile Examples status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/compile-examples.yml"]
 image:https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml/badge.svg["Spell Check status", link="https://github.com/{repository-owner}/{repository-name}/actions/workflows/spell-check.yml"]
 
-This library allows the control and configuration of the BQ24195 PMIC used on various Arduino boards.
+This library allows the control and configuration of the BQ24195 PMIC as part of the following Arduino boards:
+
+* The following MKR boards
+** MKR WIFI 1010 (SKU: ABX00023)
+** MKR FOX 1200 (SKU: ABX00014)
```

Comment:
```suggestion
```
This board does not use a BQ24195 PMIC:

https://docs.arduino.cc/hardware/mkr-fox-1200#resources